tests/e2e/utils/node_utils.go

```diff
@@ -252,6 +271,47 @@ func WaitAutoScaleNodes(cs clientset.Interface, targetNodeCount int, isScaleDown
 	return err
 }
 
+// HoldAutoScaleNodes validate node count to not change for few minutes
+func HoldAutoScaleNodes(cs clientset.Interface, targetNodeCount int) error {
+	Logf(fmt.Sprintf("checking node count stability... Target node count: %v", targetNodeCount))
+	var nodes []v1.Node
+	var err error
+	poll := 60 * time.Second
+	checkDuration := 5 * time.Minute
+	nodeConditions := map[string][]v1.NodeCondition{}
+	if err = wait.PollImmediate(poll, checkDuration, func() (bool, error) {
+		nodes, err = GetAgentNodes(cs)
+		if err != nil {
+			if IsRetryableAPIError(err) {
+				return false, nil
+			}
+			return false, err
+		}
+		if nodes == nil {
+			err = fmt.Errorf("Unexpected nil node list")
+			return false, err
+		}
+		nodeConditions = map[string][]v1.NodeCondition{}
+		for _, node := range nodes {
+			nodeConditions[node.Name] = node.Status.Conditions
+		}
+
+		Logf("Detect %v nodes, target hold %v", len(nodes), targetNodeCount)
```

Comment:
I don't think this line of log is necessary. L276 has provided the node count. If failed, L302 will tell us.